src/topMenu/TopMenuVilkarBanner.js

```diff
@@ -0,0 +1,36 @@
+import {useState} from "react";
+import * as React from 'react';
+import { Knapp } from 'pam-frontend-knapper';
+import {Link} from "react-router-dom";
+
+const TopMenuVilkarBanner = () => {
+    const [showBanner, setShowBanner] =
+        useState(localStorage.getItem('topMenuVilkarBanner') == null);
+
+    const onClose = () => {
+        setShowBanner(false);
+        localStorage.setItem('topMenuVilkarBanner', 'true');
+    };
+
+    if (!showBanner) {
+        return null;
+    }
+
+    return (
+        <div className="TopMenuVilkarBanner">
+            <div className="TopMenuVilkarBanner__content">
+                <div className="TopMenuVilkarBanner__content--container">
+                    <p>Vi har oppdatert vilkårene for å bruke tjenestene.&nbsp;
+                        <Link to="/personinnstillinger" className="lenke typo-element">
```

Comment:
Samme her, bruk className="link" for arbeidsplassen styling

Reply:
Ser nå på skissa at der er fargen svart, men du kan jo dobbeltsjekke med design om den skal være det?